# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/build/nsLayoutStatics.cpp
# Commit: 955256297d6d
# Full Hash: 955256297d6dc83d52eb77a202c9c320622c0d13
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2019-11-05 16:16:12
# Regressor Bug: 1579367
# File Overlap Count: 1
# Description:
#   Bug 1579367 - Initialize XPCJSContext explicitly, after loading user prefs. r=kmag
#   
#   This way we get the correct values for start-up prefs in the parent process.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D51061
# ==============================================================================

diff -r 68e7fb2885b0 -r 955256297d6d layout/build/nsLayoutStatics.cpp
--- a/layout/build/nsLayoutStatics.cpp	Tue Nov 05 08:19:10 2019 +0200
+++ b/layout/build/nsLayoutStatics.cpp	Tue Nov 05 06:29:13 2019 +0000
@@ -164,7 +164,6 @@
 #endif
 
   StartupJSEnvironment();
-  nsJSContext::EnsureStatics();
 
   nsGlobalWindowInner::Init();
   nsGlobalWindowOuter::Init();